packages/portfolio-deploy/src/portfolio-start.core.js

```diff
@@ -106,7 +109,24 @@ harden(makePrivateArgs);
  * @returns {Promise<void>}
  */
 export const startPortfolio = async (permitted, configStruct) => {
+  await null;
   trace('startPortfolio', configStruct);
+  /** @type {{ structure: { replace?: boolean } }} */
+  const options = /** @type any */ (configStruct.options);
+  const replace = options?.structure?.replace ?? false;
+
+  if (replace) {
+    const kit = await permitted.consume.ymax0Kit;
```

Comment:
re `unsettled value for "kp1161"` -- are you sure this is resolving? trace before/after?

Reply:
I'm pretty sure `evalProposal` invokes this code **in bundled form**. So you need to use `debugger` statements.

See also:
 - https://github.com/Agoric/agoric-sdk/discussions/9282

here's hoping for a shorter version with the relevant details.